config/cluster-baremetal-operator/cluster-baremetal-operator.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cluster-baremetal-operator
+  name: operator
```

Comment:
Currently when this command, we see:
[stack@osp-shiftstack-05 dev-scripts]$ oc get deployments -n openshift-machine-api
NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
cluster-autoscaler-operator   1/1     1            1           133m
cluster-baremetal-operator    1/1     1            1           133m
machine-api-controllers       1/1     1            1           120m
machine-api-operator          1/1     1            1           133m
metal3                        1/1     1            1           121m

After this change, will we see "operator" instead of "cluster-baremetal-operator" when we run this command?

Reply:
I wanted to make sure the naming is consistent by using "namePrefex=cluster-baremetal-" in config/profiles/default/kustomization.yaml
so the result is exactly the same